### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by waqaskanju**
In this pull:
Styling is corrected to match the design.
Unnecessary items are removed.

Preview:
image

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -17,8 +17,7 @@ const AddNewBook = () => {
     // eslint-disable-next-line camelcase
```
Comment (Ismailco) [inline src/components/AddBook.js]: - [ ] Kindly consider fixing all linter errors and try not to ignore any linters they are in place to improve your code

File: src/components/AddBook.js [hunk 2]
```diff
@@ -17,8 +17,7 @@ const AddNewBook = () => {
...
     const item_id = Math.random();
     const category = 'Crime';
-    const author = 'A.k Khan';
-    /*     dispatch(addBook(id, title, author)); */
+    const author = 'A.K Khan';
     dispatch(addBooksAsync({
       // eslint-disable-next-line camelcase
```
Comment (Ismailco) [inline src/components/AddBook.js]: - [ ] Kindly remove the commented code on line `3`, and any other commented code for cleaner code and better readability

Comment (Ismailco) [inline src/components/AddBook.js]: - [ ] Your design is good however kindly consider removing the outer white space, so your page is taking the whole space

|Your design|How it should be|
|-|-|
|Screenshot_20220715_175655|Screenshot_20220715_180244|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @waqaskanju, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally
✔️ Great design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.js (hunk 1 above)
Comment (waqaskanju) [reply src/components/AddBook.js]: This is the requirement of the Book api to send "item_id". 
image

Comment (waqaskanju) [reply src/components/AddBook.js]: if I change it to id, which is some thing not understand by api. to it does not work then.
image

File: src/components/AddBook.js (hunk 2 above)
Comment (waqaskanju) [reply src/components/AddBook.js]: Thanks

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @waqaskanju , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well implemented requested changes.

### Optional suggestions :arrow_down_small: 

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**